modules/test/playwright/tests/frontend-editor-ckeditor-web/tests/ckeditor5/classic.spec.ts

```diff
@@ -113,10 +150,75 @@ test(
 				.click();
 
 			await expect(
-				page.locator(
-					'.ck-editor__editable img[src="/documents/d/guest/moon-png"]'
+				classicPage.editable.locator(
+					'img[src="/documents/d/guest/moon-png"]'
 				)
 			).toBeVisible();
 		});
+
+		await test.step('Add a video from video modal URL input', async () => {
+			await page.reload();
+
+			await classicPage.toolbar
+				.getByRole('button', {name: 'Video'})
+				.click();
+
+			const itemSelectorFrame = classicPage.itemSelectorFrame;
+
+			const videoURLInput = itemSelectorFrame.getByLabel('Video URL');
+
+			await expect(videoURLInput).toBeEnabled();
+
+			const addButton = itemSelectorFrame.getByRole('button', {
+				exact: true,
+				name: 'Add',
+			});
+
+			await expect(addButton).toBeDisabled();
+
+			await videoURLInput.fill(sampleVideoURL);
+
+			await expect(addButton).toBeEnabled();
+
+			await page.waitForTimeout(2000);
+
+			await addButton.click();
+
+			await expect(
+				classicPage.editable
+					.frameLocator('iframe')
+					.getByLabel('YouTube Video Player')
+			).toBeVisible();
+		});
+
+		await test.step('Select a video from document library', async () => {
+			await page.reload();
+
+			await classicPage.toolbar
+				.getByRole('button', {name: 'Video'})
+				.click();
+
+			const itemSelectorFrame = classicPage.itemSelectorFrame;
+
+			await itemSelectorFrame
+				.getByRole('link', {name: 'Documents and Media'})
+				.click();
+
+			const fileCard = itemSelectorFrame.locator(
+				`.file-card[data-title="${sampleVideoTitle}"]`
+			);
+
+			await expect(fileCard).toBeVisible();
+
+			await expect(fileCard).toBeEnabled();
+
+			await fileCard.click();
+
+			await expect(
+				classicPage.editable
+					.frameLocator('iframe')
+					.getByLabel('YouTube Video Player')
+			).toBeVisible();
+		});
```

Comment:
This is the step I cannot explain why is not working locally :disappointed: 

It might be an issue just for me, let's see what CI has to say...